modules/alloy/src/main/ts/ephox/alloy/positioning/mode/ContentAnchorCommon.ts

```diff
@@ -9,19 +9,9 @@ import * as Origins from '../layout/Origins';
 import { Anchoring, NodeAnchor, nu as NuAnchor, SelectionAnchor } from './Anchoring';
 import * as AnchorLayouts from './AnchorLayouts';
 
-const capRect = (left: number, top: number, width: number, height: number): Optional<Boxes.BoxByPoint> => {
-  let newLeft = left, newTop = top, newWidth = width, newHeight = height;
-  // Try to prevent the context toolbar from getting above the editor toolbar
-  if (left < 0) {
-    newLeft = 0;
-    newWidth = width + left;
-  }
-  if (top < 0) {
-    newTop = 0;
-    newHeight = height + top;
-  }
-  const point = CssPosition.screen(SugarPosition(newLeft, newTop));
-  return Optional.some(Boxes.pointed(point, newWidth, newHeight));
+const getBox = (left: number, top: number, width: number, height: number): Optional<Boxes.BoxByPoint> => {
```

Comment:
Having looked at this, I think the capping was the wrong way to go about it, but I could be wrong. It seems to be introduced so that context toolbars wouldn't appear on top of the toolbar, but I think there are better approaches. I've removed it in this spike and handled the pinning / docking / capping in a different way.

Reply:
I'd agree here, when you linked this the other day I had the same thought.